src/test/k6/requests/requests.js

```diff
@@ -24,7 +24,7 @@ const request = function (method, endpoint, authToken, body, params) {
         {
             method: method,
             url: url,
-            body: body ? JSON.stringify(body) : null,
+            body: formData ? formData.body() : body ? JSON.stringify(body) : null,
```

Comment:
I find the double `?:` very hard to read. Can you refactor that a bit? E.g.,

```typescript
const requestBody = formData ? formData.body() : JSON.stringify(body);
```
```suggestion
            body: requestBody ? requestBody : null,
```

Reply:
I believe that this change would change the semantics:
If the body is `null`, `JSON.stringify(body)` would be `"null"` and then the `requestBody` will always be truthy

Reply:
Indeed, I didn’t think too hard about it. Then maybe some proper if/else-s are needed to represent the same logic.

Reply:
I've adjusted it now.